src/blueprints/Do.Blueprints.Service/Core/Dotnet/DotnetSystem.cs

```diff
@@ -0,0 +1,11 @@
+namespace Do.Blueprints.Service.Core.Dotnet;
+
+public class DotnetSystem : ISystem
+{
+    public DateTime Now => DateTime.Now;
+}
+
+public interface ISystem
+{
+    DateTime Now { get; }
+}
```

Comment:
move `ISystem` to its own file directly under `/Core` folder. namespace should be `Do.Core`